rts/lra/client/src/main/java/io/narayana/lra/client/NarayanaLRAClient.java

```diff
@@ -341,12 +344,14 @@ public URI startLRA(URI parentLRA, String clientID, Long timeout, ChronoUnit uni
 
             return lra;
         } catch (UnsupportedEncodingException uee) {
+            String logMsg = LRALogger.i18nLogger.error_invalidFormatToEncodeParentUri(parentLRA, uee);
+            String errorMsg = String.format("%s (%s:%s %s %s %s (%s:%s) )", logMsg, "Cannot connect to the LRA coordinator", coordinatorUrl,
```

Comment:
Incorrect usage of String.format. The syntax is defined in the javadoc for Formatter: https://docs.oracle.com/javase/8/docs/api/java/util/Formatter.html